src/scripts/install.py

```diff
@@ -98,12 +111,14 @@ def main(args = None):
         exe_mode &= (umask ^ 0o777)
 
     def copy_file(src, dst):
+        dst = prepend_destdir(dst)
```

Comment:
same here, prepend destdir in caller. This makes the strange return value obsolete